stories/atoms/components/typography/Typography.stories.mdx

```diff
@@ -137,27 +143,45 @@ There is no default size to preserve what's being ultimately rendered.
 
 ## Colors
 
-The different colors available are: `text` (default), `highlighted-text`, `info`, `success`, `warning`, `error` which
+The different colors available are: `text` (default), `highlighted-text`, `inverted-text`, `info`, `success`, `warning`, `error`, `primary-invariant`, `secondary-invariant` which
```

Comment:
I think it would be nice to explain in a few words the differences between variant, invariant and inverted. 
It may be clear to us but not necessarily to a new user of the library.
This would allow the user to see more clearly which color to use according to his needs, if he wants it to change or not according to the theme... What do you think about it?

Reply:
I thought that the visualization of the color would be enough without further explanation so as not to influence the user. But other opinions are welcome

Reply:
Agreeing with @lolottetheclash 's point, I think there is a need to better document the colors used for the Typography component.

For this, it would be necessary, in the same `colors` story, to:
- dissociate "basic" colors from those with a deeper purpose such as `invariants` or `inverted` ones.
- avoid offering colors on a background that makes them unreadable.
- document complex colors such as inversions in order to explain to the user where to use them and especially how to use them (in which frame? In which component?)

This is all the more important when the themes are complex, I think we have to be more rigorous on this point.